README.markdown

```diff
@@ -30,7 +30,8 @@ or Wget:
 
     wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.29.0/install.sh | bash
 
-<sub>The script clones the nvm repository to `~/.nvm` and adds the source line to your profile (`~/.bash_profile`, `~/.zshrc` or `~/.profile`).</sub>
+<sub>The script clones the nvm repository to `~/.nvm` and adds the source line to your profile (`~/.bash_profile`, `~/.zshrc` or `~/.profile`).</sub> 
+Eg. `source ~/.profile`
```

Comment:
And you need to kown, Install script && Manual install are two different install ways. Since I choose the 'Install script' and use it, how could i read the 'Manual install'.